prime-router/src/main/kotlin/azure/WorkflowEngine.kt

```diff
@@ -245,16 +249,13 @@ class WorkflowEngine(
                 receivingFacility
             )
         } catch (ex: Exception) {
-            context?.logger?.warning(
+            logger.error(
                 "Got exception while dispatching to schema ${report.schema.name}" +
                     ", and rcvr ${receiver.fullName}"
```

Comment:
I know this isn't under change right now, why not use 'receiver' instead of 'rcvr'?